tailwind.config.mjs

```
@@ -8,6 +8,37 @@ export default {
     extend: {
       colors: {
+        /** color token */
+        primary: {
+          700: '#0C1B71',
+          600: '#2238B3',
+          500: '#4A66FF',
+          400: '#788DFF',
+          300: '#8799FF',
+          200: '#B1BDFF',
+          100: '#DDE2FF',
+        },
```

Biome (2.4.14)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)